chain/indexer/src/streamer/types.rs

```diff
@@ -39,7 +39,7 @@ pub struct IndexerExecutionOutcomeWithReceipt {
 
 #[derive(Debug, Serialize, Deserialize)]
 pub struct IndexerShard {
-    pub shard_id: types::ShardId,
+    pub shard_ord: types::ShardOrd,
```

Comment:
@khorolets @frol would the change here break anything? Looks like the struct derives `Serialize`